Cargo.toml

```diff
@@ -64,6 +65,8 @@ indexmap = "1.6.1"
 notify-rust = { version = "4.2.2", optional = true }
 semver = "0.11.0"
 which = "4.0.2"
+shadow-rs = "0.5"
```

Comment:
Is there a reason not to use the patch version as well? We recently changed all our `Cargo.toml` entries to use `X.Y.Z` numbers to avoid certain dependency issues--would be nice if we could do that here.

Reply:
Done.